pyproject.toml

```diff
@@ -54,3 +57,66 @@ strict_concatenate = true
 [tool.bandit]
 tests = ["B402","B401","B310","B403","B404","B302","B321","B301","B312","B303","B306","B304","B305","B308","B307","B313","B320","B405","B409","B408","B407","B406","B410","B411","B101","B102","B201","B104","B107","B106","B105","B608","B108","B701","B609","B601","B501","B103","B507","B502","B504","B605","B606","B607","B602","B603","B112","B110","B702","B505","B506", "B323", "B610", "B703", "B611", "B412","B413"]
 exclude_dirs = ["./test/*"]
+
+[tool.pylint.main]
+exit-zero = true
+jobs = 0
+recursive = true
+suggestion-mode = true
+# ignore-paths =
+
+[tool.pylint.basic]
+class-attribute-naming-style = "snake_case"
+
+[tool.pylint.classes]
+defining-attr-methods = ["__init__", "__new__", "__post_init__"]
+
+[tool.pylint.imports]
+# Deprecated modules which should not be used, separated by a comma.
+# deprecated-modules =
+
+# Force import order to recognize a module as part of the standard compatibility
+# libraries.
+# known-standard-library =
+
+[tool.pylint.logging]
+logging-format-style = "new"
+
+disable = [
+    "raw-checker-failed",
+    "bad-inline-option",
+    "locally-disabled",
+    "file-ignored",
+    "suppressed-message",
+    "useless-suppression",
+    "deprecated-pragma",
+    "use-symbolic-message-instead",
+    "design",
+    "line-too-long",
+    "too-many-lines",
+    "too-many-nested-blocks",
+    "typecheck",
+    "import-error",
+    "trailing-whitespace"
+]
+
+[tool.pylint.method_args]
+# List of qualified names (i.e., library.method) which require a timeout
+# parameter e.g. 'requests.api.get,requests.api.post'
+timeout-methods = ["requests.api.delete", "requests.api.get", "requests.api.head", "requests.api.options", "requests.api.patch", "requests.api.post", "requests.api.put", "requests.api.request"]
+
+[tool.pylint.reports]
+reports = false # TODO find a clean way to parametrise this.
+
+[tool.pylint.spelling]
+max-spelling-suggestions = 60
+
+# List of comma separated words that should not be checked.
+# spelling-ignore-words =
+
+[tool.pylint.string]
+check-quote-consistency = true
+check-str-concat-over-line-jumps = true
+
+[tool.pylint.variables]
+init-import = true
```

Comment:
I don't think we should check for unused imports in `__init__` files. In my understanding, those files are for exposing certains parts of a module.
Certain parts could be exported as public API without being used right now. 

In my opinion, public API should be exported even if unused because someone else needing that later might not know the module and what is supposed to be public, thus exporting something that should not be.

Reply:
Hi, 
Your concern is legitimate. Public API should be available and exported even if the module that declared the API didn't use it. Hopefully we do not need to import the API in the `__init__` file for this to append. An empty file does this job wonderfully. 
I checked the official [documentation](https://docs.python.org/3/tutorial/modules.html) and did not find a use case for those unused imports that could overcome the drawback of having to check for unused imports there.